SpringSchool/src/main/java/com/example/springschool/services/SchoolService.java

```diff
@@ -0,0 +1,37 @@
+package com.example.springschool.services;
+
+import com.example.springschool.entities.School;
+import com.example.springschool.entities.Student;
+import com.example.springschool.repositories.SchoolRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+public class SchoolService {
+    SchoolRepository schoolRepository;
+
+    public List<School> getAllSchools() {
+        return this.schoolRepository.findAll();
+    }
+
+    public School getSchoolByIdOrThrow(Long id) {
+        return this.schoolRepository.findById(id).orElseThrow();
+    }
+
+    public List<Student> getAllStudentsById(Long id) {
+        return this.schoolRepository.findById(id).orElseThrow().getStudents();
```

Comment:
Perpanaudojam turimus funkcionalumus:

```suggestion
        return this.getSchoolByIdOrThrow(id).getStudents();
```

Reply:
tiesa